prow/cmd/line/main.go

```diff
@@ -347,12 +347,14 @@ func (c *testClient) TestKubernetes() error {
 			Secret: &kube.SecretSource{
 				Name: "ssh-key-secret",
 			},
+			defaultMode: 256,
```

Comment:
it's only an issue if it's specified in a json file. by the time this value gets there it'll be converted into its integer value.